docs/Project.toml

```diff
@@ -13,12 +13,12 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-DataFrames = "0.21"
+DataFrames = "1"
 Documenter = "0.24"
 GaussianMixtures = "0.3"
 GaussianProcesses = "0.12"
 JLD = "0.9, 0.10"
-PyCall = "1.91"
+PyCall = "1.92"
 PyCallJLD = "0.2"
-PyPlot = "2.9"
-RDatasets = "0.6"
+PyPlot = ">=2.9"
+RDatasets = ">=0.6"
```

Comment:
I wonder if we could get rid of `docs/Project.toml` entirely... It's clearly not a well-maintained file.

Reply:
I think that we will always need it for the Documenter dependency. However, I did not mean to leave the bounds changes for `PyPlot` and `RDatasets`  in the PR (it doesn't hurt, but it wasn't needed). I was trying to debug the failing docs build, but it ended up being because the Julia package wasn't being built, and not an issue with the packages.

Reply:
Not a Documenter expert by any means, but looking at the docs, [it looks like we could/should get rid of most of these bounds](https://documenter.juliadocs.org/stable/man/hosting/#docs/Project.toml)

Reply:
@OkonSamuel https://github.com/cstjean/ScikitLearn.jl/commit/4149b4381b05e0c291d0847de08cc4fb0928f0a8 created the Documenter code. Do you remember if you had a specific reason for adding all these dependencies?